schema.sql

```diff
@@ -1,51 +1,57 @@
-CREATE TABLE animals (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    date_of_birth DATE,
-    escape_attempts INT,
-    neutered BOOLEAN,
-    weight_kg DECIMAL(5, 2)
+CREATE TABLE owners(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(20),
+  age INT,
+  PRIMARY KEY(id)
 );
 
-ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
-
-CREATE TABLE owners (
-    id SERIAL PRIMARY KEY,
-    full_name VARCHAR(255),
-    age INTEGER
+CREATE TABLE species(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  PRIMARY KEY(id)
 );
 
-CREATE TABLE species (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255)
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  date_of_birth DATE,
+  escape_attempts INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+  species_id INT REFERENCES species(id),
+  owner_id INT REFERENCES owners(id),
+  PRIMARY KEY(id)
 );
 
-ALTER TABLE animals
-    DROP COLUMN species,
-    ADD COLUMN species_id INTEGER,
-    ADD COLUMN owner_id INTEGER;
-
-ALTER TABLE animals
-    ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id),
-    ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
-
-CREATE TABLE vets (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    age INTEGER,
-    date_of_graduation DATE
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
 );
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍